.sync.yml

```diff
@@ -7,11 +7,6 @@ Gemfile:
   extra:
     - gem: webmock
```

Comment:
https://github.com/bblimke/webmock only lists MRI 2.0 and higher. Soon we'll drop Puppet 3 support and that means we can drop Ruby 1.9 as well since the PL AIO install includes a ruby 2+ (for better or worse). Should we address webmock then?